x/evidence/simulation/genesis.go

```diff
@@ -0,0 +1,55 @@
+package simulation
+
+// DONTCOVER
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/evidence/exported"
+	"github.com/cosmos/cosmos-sdk/x/evidence/types"
+)
+
+// Simulation parameter constants
+const (
+	Evidence = "evidence"
+)
+
+// GenEvidences randomized Evidence
+func GenEvidences(r *rand.Rand, accs []simtypes.Account) []exported.Evidence {
+	totalEv := r.Intn(20)
+	evidence := make([]exported.Evidence, totalEv)
+
+	for i := 0; i < totalEv; i++ {
+		simacc, _ := simtypes.RandomAcc(r, accs)
+		evidence[i] = &types.Equivocation{
```

Comment:
Is this the only type of evidence we have at the moment?

Reply:
Also, I feel like it's a bit odd to generate evidence with heights, times, and powers that are not real (i.e. corresponding to an actual validator height, time, and power).

Reply:
I don't really see the purpose of the PR IMHO.

Reply:
ah yeah, so when the underlying issue was created we didn't have any evidence but `Equivocation`. Now that IBC is on master I can try simulating client misbehaviors. 

I'll put this PR in draft mode again

Reply:
> Now that IBC is on master I can try simulating client misbehaviors.

So I tried to do this but it'll require to add ICS02 and ICS07 simulations to create clients and submit evidence. I'd propose to merge the PR with just the `DecodeStore` changes and add an empty genesis state (default) for the evidence module for now